.github/workflows/lint-test.yml

```diff
@@ -6,22 +6,43 @@ jobs:
   lint-test:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v1
+      - uses: actions/checkout@v4
```

Comment:
We should probably also introduce `dependabot.yml` to make sure these get updated automatically: https://github.com/adfinis/helm-charts/blob/main/.github/dependabot.yml

Reply:
`done`